owners/src/owners_bot.js

```diff
@@ -125,6 +129,34 @@ class OwnersBot {
     await this.runOwnersCheck(github, pr);
   }
 
+  /**
+   * Adds a comment tagging always-notify owners of changed files.
+   *
+   * @param {!GitHub} github GitHub API interface.
+   * @param {!number} prNumber pull request number.
+   * @param {!FileTreeMap} fileTreeMap map from filenames to ownership subtrees.
+   */
+  async createNotifications(github, prNumber, fileTreeMap) {
+    const botComments = await github.getBotComments(prNumber);
+    if (botComments.length) {
+      // Avoid adding duplicate notification comment.
+      return;
+    }
+
+    const notifies = this._getNotifies(fileTreeMap);
+
+    const tag = name => `@${name}`;
+    const header = 'Some owners are notified of changes to files in this PR:';
+    const fileNotifyComments = Object.entries(notifies).map(
+      ([filename, names]) => `- ${filename}: ${names.map(tag).join(', ')}`
```

Comment:
Can we update the comment body to be more user friendly, and only mention an owner once? Something like 
```
Hey owner1, these files were changed
- file a
- file b

Hey owner2, these files were changed:
- file c

Reply:
Soitenly! Done.